modules/nodes-control-plane-osp-migrating.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * TBD
+
+:_content-type: PROCEDURE
+[id="nodes-control-plane-osp-migrating_{context}"]
+= Migrating control plane nodes from one RHOSP host to another
+
+You can run a script that moves a control plane node from one {rh-openstack-first} node to another.
+
+.Prerequisites
+
+* The environment variable `OS_CLOUD` refers to a `clouds` entry that has administrative credentials in a `clouds.yaml` file.
+* The environment variable `KUBECONFIG` refers to a configuration that contains administrative {product-title} credentials.
+
+.Procedure
+
+* From a command line, run the following script:
+
+[source,bash]
+----
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+if [ $# -lt 1 ]; then
+	echo "Usage: '$0 node_name'"
+	exit 64
+fi
+
+# Check for admin OpenStack credentials
+openstack server list --all-projects >/dev/null || { >&2 echo "The script needs OpenStack admin credentials. Exiting"; exit 77; }
+
+# Check for admin OpenShift credentials
+oc adm top node >/dev/null || { >&2 echo "The script needs OpenShift admin credentials. Exiting"; exit 77; }
+
+set -x
+
+declare -r node_name="$1"
+declare server_id
+server_id="$(openstack server list --all-projects -f value -c ID -c Name | grep "$node_name" | cut -d' ' -f1)"
+readonly server_id
+
+# Drain the node
+oc adm cordon "$node_name"
+oc adm drain "$node_name" --delete-emptydir-data --ignore-daemonsets
```

Comment:
We might have to use the `--force` option in the drain command due to bug https://bugzilla.redhat.com/show_bug.cgi?id=2047257. I'll let @pierreprinetti sort it out next week when he's back (and I'm off).

Reply:
cf https://github.com/openshift/installer/pull/5618

Reply:
```suggestion
oc adm drain "$node_name" --delete-emptydir-data --ignore-daemonsets --force
```

All good? @mandre @pierreprinetti

Reply:
Yes please add the `--force` flag, it's not completely clear to me at the moment why we need it -- it looks like the `guard` pods are a late addition in 4.10 and will not drain properly unless passing the `--force` flag.

We'll investigate in 4.11 and possibly remove this flag in the future once we understand the issue better.